src/components/admin/Login.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <div class="container login-container">
+        <Dialog/>
+        <div class="row">
+            <div class="col-md-6 login-form-1">
+                <h3>{{$t('login.title')}}</h3>
+                <form @submit.prevent="handleSubmit">
+                    <div class="form-group">
+                        <input id="username" type="text" class="form-control" placeholder="Twój login" value=""/>
+                    </div>
+                    <div class="form-group">
+                        <input id="password" type="password" class="form-control" placeholder="Twoje hasło"
+                               value=""/>
+                    </div>
+                    <div class="form-group">
+                        <input type="submit" class="btnSubmit" value="Zaloguj się"/>
+                    </div>
+                    <div class="form-group">
+                        <a href="#" class="ForgetPwd">{{$t('login.forgetPassword')}}</a>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+</template>
+<script>
+
+    import Dialog from "../common/Dialog";
+
+    export default {
+        name: 'login',
+        components: {Dialog},
+        data() {
+            return {}
+        },
+        methods: {
+            handleSubmit() {
+                const login = document.getElementById('username').value;
+                const password = document.getElementById('password').value;
+                const authorization = `Basic ${btoa(`${login}:${password}`)}`;
+                fetch("http://jakas.pl"/*Todo endpoint from backend*/, {
+                    method: "GET",
+                    headers: {
+                        Authorization: authorization
+                    }
+                }).then(() => {
+                    sessionStorage.setItem("Authorization", authorization);
+                    window.location.href = '/admin/home';
+                })
+                    .catch(() => {
+                        //TODO add error to warning message
+                        this.$root.$emit('showDanger','Podano nie prawidłowe dane logowania')
+                    });
+            },
+        }
+    }
+</script>
+<style>
+    .login-container {
```

Comment:
Jeżeli będziemy używać tych stylów w innych miejsach, to przenieśmy do style.css, jeżeli tylko tuttaj to dodać tag scoped: https://vue-loader.vuejs.org/guide/scoped-css.html